### ISSUE TITLE
Icon Request: Checklist

### ISSUE THREAD

**Issue by derpmarine168** [feathericons/feather#504]
## Icon Request

* Icon name: Checklist
* Use case: WYSIWYG editors
* Screenshots of similar icons:

![img_9489](https://user-images.githubusercontent.com/28905212/48950405-ee498900-ef32-11e8-9a61-808ea26eaec0.jpeg) From github editor

**Comment by ahtohbi4** [comment]
What about these ones?

<img width="913" alt="Снимок экрана 2019-07-01 в 16 14 18" src="https://user-images.githubusercontent.com/3686790/60439377-6c81b280-9c1b-11e9-8076-c424514bb8ef.png">

<img width="542" alt="Снимок экрана 2019-07-01 в 16 14 45" src="https://user-images.githubusercontent.com/3686790/60439381-70153980-9c1b-11e9-8ec0-54a78f2d7059.png">

**Comment by johnletey** [comment]
@ahtohbi4 Is there a reason why the numbers in the ordered-list are unreadable? Love the icons though!

**Comment by locness3** [comment]
> @ahtohbi4 Is there a reason why the numbers in the ordered-list are unreadable?

Yes, that's because it's hard to do small details with 2px strokes.
